packages/dashmate/test/e2e/lib/manageDockerData.js

```diff
@@ -0,0 +1,78 @@
+const { getConfig } = require('./manageConfig');
+const { SERVICES } = require('./constants/services');
+
+/**
+ * Remove docker volumes
+ * @param {string} configName
+ * @param {object} dockerContainer
+ */
+async function removeVolumes(configName, dockerContainer) {
+  const config = await getConfig(configName);
+  const { COMPOSE_PROJECT_NAME: projectName } = config.toEnvs();
+  const projectVolumeNames = await dockerContainer.getVolumeNames(config.toEnvs());
+
+  projectVolumeNames
+    .map((volumeName) => `${projectName}_${volumeName}`)
+    .map(async (volumeName) => await dockerContainer.getVolume(volumeName).remove());
```

Comment:
You are missing await Promise.all() on this function

Reply:
Fixed, wait for next commit